libs/application-generic/package.json

```diff
@@ -83,6 +83,8 @@
     "launchdarkly-node-server-sdk": "^7.0.1",
     "lodash": "^4.17.15",
     "mixpanel": "^0.17.0",
+    "nanoid": "^3.1.20",
+    "nestjs-graceful-shutdown": "^1.0.3",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** `nestjs-graceful-shutdown` appears to be a newly added dependency. Was this intentional?‏